static/src/js/components/Timeline/Timeline.js

```diff
@@ -565,9 +587,10 @@ class Timeline extends Component {
   }
 
   render() {
+    const { timeline: { isOpen } } = this.props
     return (
-      <section className="timeline">
-        <div ref={(el) => { this.el = el }} />
+      <section className={isOpen ? 'timeline' : 'hidden'}>
```

Comment:
We prefer using the `classNames` library for this over a ternary. It would look something like this:

```
const timelineClasses = classNames([
  'timeline',
  {
    'timeline--is-hidden': !isOpen
  }
])

return (
  <section className={timelineClasses}>
```